packages/cli/src/upload/upload.js

```diff
@@ -72,7 +72,7 @@ function getCurrentHash() {
   if (envVars.TRAVIS_PULL_REQUEST_SHA) return envVars.TRAVIS_PULL_REQUEST_SHA;
   if (envVars.TRAVIS_COMMIT) return envVars.TRAVIS_COMMIT;
 
-  const result = childProcess.spawnSync('git', ['rev-list', '--no-merges', '-n', '1', 'HEAD'], {
```

Comment:
does this impact things or just readability preference?

Reply:
fwiw i find the new change to be more readable. one arg per array entry is good